core/src/SYCL/Kokkos_SYCL_Parallel_Reduce.hpp

```diff
@@ -49,6 +49,7 @@
 
 #include <vector>
 #if defined(KOKKOS_ENABLE_SYCL)
+#include <Kokkos_Parallel_Reduce.hpp>
```

Comment:
hu?

Reply:
I didn't dive too deep into why I need the header for the Makefile build but not for the `CMake` build. It still makes sense to include the header IMHO.